raw-data/PH/phoible_inventories.tsv

```diff
@@ -15409,27 +15409,27 @@
 				"ʃ"				
 				"x"				
 				"h"				
-				"sʼ"				
-				"ʃʼ"				
-				"xʼ"				
+				"sʼ"	"""Subsequent literature (by Wash 2001) suggests that glottalized means ejective for Beeler based on Beeler 1976 mentioning a 'staccato effect'."""			
+				"ʃʼ"	"""Subsequent literature (by Wash 2001) suggests that glottalized means ejective for Beeler based on Beeler 1976 mentioning a 'staccato effect'."""			
+				"xʼ"	"""Subsequent literature (by Wash 2001) suggests that glottalized means ejective for Beeler based on Beeler 1976 mentioning a 'staccato effect'."""			
 				"sʰ"				
 				"ʃʰ"				
 				"ts"				
 				"t̠ʃ"				
-				"tsʼ"				
-				"t̠ʃʼ"				
+				"tsʼ"	"""Subsequent literature (by Wash 2001) suggests that glottalized means ejective for Beeler based on Beeler 1976 mentioning a 'staccato effect'."""			
+				"t̠ʃʼ"	"""Subsequent literature (by Wash 2001) suggests that glottalized means ejective for Beeler based on Beeler 1976 mentioning a 'staccato effect'."""			
 				"tsʰ"				
 				"t̠ʃʰ"				
 				"l"				
 				"j"				
 				"w"				
-				"l̰"				
-				"j̰"				
-				"w̰"				
+				"ˀl"				
+				"ˀj"				
+				"ˀw"				
 				"m"				
 				"n"				
-				"m̰"				
-				"n̰"				
+				"ˀm"				
+				"ˀn"				
```

Comment:
These aren't described as "pre-glottalized" so the diacritic should be on the right of the base glyph.